compiler/src/iree/compiler/Dialect/HAL/Transforms/ResolveEntryPointOrdinals.cpp

```diff
@@ -23,20 +23,20 @@ class ResolveCommandBufferDispatchOrdinals
                                 PatternRewriter &rewriter) const override {
     auto symbol = SymbolTable::lookupNearestSymbolFrom(op, op.entry_point());
     assert(symbol && "missing ExecutableEntryPoint symbol");
-    auto entryPointOp = cast<IREE::HAL::ExecutableEntryPointOp>(symbol);
+    auto exportOp = cast<IREE::HAL::ExecutableExportOp>(symbol);
```

Comment:
Can you also rename this pass/file to ResolveExportOrdinals? 

I searched the repo for other passes that should be renamed and I think https://github.com/google/iree/blob/main/compiler/src/iree/compiler/Bindings/Native/Transforms/WrapEntryPoints.cpp doesn't need updating.

Reply:
good catch!